scripts/setup

```diff
@@ -23,6 +23,14 @@ then
         mkdir -p "${HOME}/.aws"
         mkdir -p "nginx/srv/dist/"
 
+        AWS_PROFILE=${AWS_PROFILE:-raster-foundry} \
```

Comment:
Might be able to combine these with `bash -c "./scripts/bootstrap --env && ./scripts/bootstrap --sbtopts"` since multiple command line flags doesn't work with `bootstrap`.